apps/mongodb/liveness/mongo_liveness.yml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: liveness-mongo
+ 
+spec:
+  template:
+    metadata:
+      name: liveness
+   
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: liveness
+        image: ranjnshashank855/py-server:t2
```

Comment:
can we have this image in the openebs repo ?

Reply:
@ksatchit Sure sir